### ISSUE TITLE
Hash mismatch for github.copilot-chat vscode extension

### ISSUE THREAD

**Issue by WeetHet** [NixOS/nixpkgs#295641]
https://github.com/NixOS/nixpkgs/blob/a0906f14161a5c5792e9883117b9471f5bf6df72/pkgs/applications/editors/vscode/extensions/default.nix#L1870

I live on nixpkgs-unstable and this gives me an error:
```
error: hash mismatch in fixed-output derivation '/nix/store/y3wzvsymir60ydkrw81s4mndplfxjgvv-github-copilot-chat.zip.drv':
         specified: sha256-4ArWVFko2T6ze/i+HTdXAioWC7euWCycDsQxFTrEtUw=
            got:    sha256-v9RS+pEEQzq2t0EY/gVTTfIwRlGmIuIq2qCockDpsn8=
```
cc @Laurent2916

**Comment by Laurent2916** [comment]
damn sorry about that, I reviewed https://github.com/NixOS/nixpkgs/pull/294690 too quickly, didn't saw the hash hadn't been updated. Also I looked at my shell history, I did try to build it, though with the wrong nixpkgs revision :sweat: (I ran `NIXPKGS_ALLOW_UNFREE=1 nix build -L NixOS/nixpkgs#vscode-extensions.github.copilot-chat --impure` instead of `NIXPKGS_ALLOW_UNFREE=1 nix build -L github:NixOS/nixpkgs#vscode-extensions.github.copilot-chat --impure`). I'll submit a fix!
